### PR TITLE
feat: Task Assigning Quickstart

### LINKED ISSUE #385: Task Assigning quickstart

### PR DESCRIPTION
**Pull request by zepfred**
## Description of the change

This PR adds a new quickstart for the Task Assigning problem.

[Screencast from 2024-04-14 19-02-37.webm](https://github.com/TimefoldAI/timefold-quickstarts/assets/11961659/cabd74a3-6cdb-4199-aa87-02e3a75e5071)

### Development

- [x] The changes have been covered with tests, if necessary.
- [x] You have a green build, with the exception of the flaky tests.
- [x] UI and JS files are fully tested, the user interface works for all modules affected by your changes (e.g., solve and analyze buttons).
- [x] The network calls work for all modules affected by your changes (e.g., solving a problem).
- [x] The console messages are validated for all modules affected by your changes.

### Code Review

- [x] This pull request includes an explanatory title and description.
- [x] The GitHub issue is linked.
- [x] At least one other engineer has approved the changes.
- [ ] After PR is merged, inform the reporter.

### CODE REVIEW

File: use-cases/task-assigning/src/main/java/org/acme/taskassigning/domain/TaskAssigningSolution.java [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    private List<Customer> customers;
+
+    @ValueRangeProvider
+    @PlanningEntityCollectionProperty
+    private List<Task> tasks;
+
+    @PlanningEntityCollectionProperty
+    private List<Employee> employees;
+
+    @PlanningScore(bendableHardLevelsSize = 1, bendableSoftLevelsSize = 5)
```
Comment (triceo) [inline use-cases/task-assigning/src/main/java/org/acme/taskassigning/domain/TaskAssigningSolution.java]: This is a bit crazy. I suggest we spend a bit of time thinking about the constraints and how they should be weighted against each other, and based on that, replace this with a normal `HardMediumSoftScore`.

**Review by triceo — CHANGES_REQUESTED** [review]
Overall, I like it!
But let's rethink the score - the `Bendable` for this usecase seems overkill. Let me know if you need help rebalancing the weights.

File: use-cases/task-assigning/src/main/java/org/acme/taskassigning/domain/TaskAssigningSolution.java (hunk 1 above)
Comment (zepfred) [reply use-cases/task-assigning/src/main/java/org/acme/taskassigning/domain/TaskAssigningSolution.java]: Our bendable score has now one hard level and three soft levels.

File: use-cases/task-assigning/pom.xml [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+
+    <groupId>org.acme</groupId>
+    <artifactId>timefold-solver-quarkus-task-assigning-quickstart</artifactId>
+    <version>1.0-SNAPSHOT</version>
+
+    <properties>
+        <maven.compiler.release>17</maven.compiler.release>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+
+        <version.io.quarkus>3.8.3</version.io.quarkus>
```
Comment (triceo) [inline use-cases/task-assigning/pom.xml]:
```suggestion
        <version.io.quarkus>3.9.3</version.io.quarkus>
```

**Review by triceo — APPROVED** [review]

**Pull request merged**